shinydashboard/lantern/server.R

```diff
@@ -1,15 +1,71 @@
 # Define server function
 function(input, output, session) {
+  fhir_version_list <- get_fhir_version_list(endpoint_export_tbl)
+  vendor_list <- get_vendor_list(endpoint_export_tbl)
 
   callModule(
     dashboard,
     "dashboard_page")
 
   callModule(
     endpointsmodule,
-    "endpoints_page")
+    "endpoints_page",
+    reactive(input$fhir_version),
+    reactive(input$vendor))
 
-   callModule(
+  callModule(
     availability,
     "availability_page")
-}
\ No newline at end of file
+
+  callModule(
+    capabilitymodule,
+    "capability_page",
+    reactive(input$fhir_version),
+    reactive(input$vendor))
+
+   page_name_list <- list("dashboard_tab" = "Current Endpoint Metrics",
+                          "endpoints_tab" = "List of Endpoints",
+                          "capability_tab" = "Capability Page",
+                          "availability_tab" = "Endpoint Server Availability",
+                          "performance_tab" = "Mean Response Time",
```

Comment:
Mean Response Time tab has been temporarily removed. Any benefit to keeping this here?

Reply:
No, it should be removed.